.github/workflows/commits.yml

```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: Conventional commit check
+on: [push]
+jobs:
+  commit_messages:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Conventional commit check
+        id: conventional_commit_check
+        run: |
+          regex="^(build|feat|fix|test|perf|refactor|docs|style)(\((advance-runner|dispatcher|graphql-server|indexer|inspect-server|state-server|auxiliary|contracts|deploy|arbitration|cli)\))?:\s.*(\n\n.*)*(\n\n(\w|[-/])+.*)*$"
```

Comment:
It would be great if we could find a GitHub actions that lets us keep a nicer list of commit types and scopes.